<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Managment Platform</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="addTab">
       <attribute name="title">
        <string>Add Books</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="singleGroupBox">
            <property name="title">
             <string>Add single book</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLabel" name="ISBNLabel">
                   <property name="text">
                    <string>&amp;ISBN:</string>
                   </property>
                   <property name="buddy">
                    <cstring>ISBNEdit</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="categoryLabel">
                   <property name="text">
                    <string>&amp;Category:</string>
                   </property>
                   <property name="buddy">
                    <cstring>categoryEdit</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="titleLabel">
                   <property name="text">
                    <string>&amp;Title:</string>
                   </property>
                   <property name="buddy">
                    <cstring>titleEdit</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="publisherLabel">
                   <property name="text">
                    <string>&amp;Publisher:</string>
                   </property>
                   <property name="buddy">
                    <cstring>publiserEdit</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="yearLabel">
                   <property name="text">
                    <string>&amp;Year:</string>
                   </property>
                   <property name="buddy">
                    <cstring>yearEdit</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="authorLabel">
                   <property name="text">
                    <string>&amp;Author:</string>
                   </property>
                   <property name="buddy">
                    <cstring>authorEdit</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="priceLabel">
                   <property name="text">
                    <string>&amp;Price:</string>
                   </property>
                   <property name="buddy">
                    <cstring>priceEdit</cstring>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QLineEdit" name="ISBNEdit"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="categoryEdit"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="titleEdit"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="publiserEdit"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="yearEdit"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="authorEdit"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="priceEdit"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="addButton">
               <property name="text">
                <string>Add IT!</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="multiGroupBox">
            <property name="title">
             <string>Add multiple books</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLineEdit" name="lineEdit">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="loadButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="addmultiButton">
               <property name="text">
                <string>Add Them!</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="queeryTab">
       <attribute name="title">
        <string>Books Query</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QLabel" name="ISBNLabel_2">
                    <property name="text">
                     <string>ISBN:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="categoryLabel_2">
                    <property name="text">
                     <string>Category:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="titleLabel_2">
                    <property name="text">
                     <string>Title:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="publisherLabel_2">
                    <property name="text">
                     <string>Publisher:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="authorLabel_2">
                    <property name="text">
                     <string>Author:</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QLineEdit" name="ISBNEdit_2"/>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="categoryEdit_2"/>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="titleEdit_2"/>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="publiserEdit_2"/>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="authorEdit_2"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Year from:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Price from:</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QLineEdit" name="yearFromEdit"/>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="priceFromEdit"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>to:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>to:</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QLineEdit" name="yearToEdit"/>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="priceToEdit"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="searchButton">
              <property name="text">
               <string>Search It!</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="queryTableView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="borrowTab">
       <attribute name="title">
        <string>Borrow Books</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>&amp;ID: </string>
                </property>
                <property name="buddy">
                 <cstring>borrowIDEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>I&amp;SBN:</string>
                </property>
                <property name="buddy">
                 <cstring>borrowISBNEdit</cstring>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QLineEdit" name="borrowIDEdit"/>
              </item>
              <item>
               <widget class="QLineEdit" name="borrowISBNEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <item>
               <widget class="QPushButton" name="conformButton">
                <property name="text">
                 <string>Comform ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="borrowbutton">
                <property name="text">
                 <string>Borrow IT!</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="borrowTableView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="returnTab">
       <attribute name="title">
        <string>Return Books</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_18">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>&amp;ID: </string>
                </property>
                <property name="buddy">
                 <cstring>returnIDEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>I&amp;SBN:</string>
                </property>
                <property name="buddy">
                 <cstring>returnISBNEdit</cstring>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_19">
              <item>
               <widget class="QLineEdit" name="returnIDEdit"/>
              </item>
              <item>
               <widget class="QLineEdit" name="returnISBNEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_20">
              <item>
               <widget class="QPushButton" name="conformButton_2">
                <property name="text">
                 <string>Comform ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="returnButton">
                <property name="text">
                 <string>Return IT!</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="returnTableView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="accountTab">
       <attribute name="title">
        <string>Accounts</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QTableView" name="accountTableView"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_21">
            <item>
             <widget class="QPushButton" name="addAccountButton">
              <property name="text">
               <string>&amp;Add an account</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteAccountButton">
              <property name="text">
               <string>D&amp;elete an account</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="confirmEditButton">
              <property name="text">
               <string>&amp;Confirm changes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cancelEditButton">
              <property name="text">
               <string>&amp;Discard changes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>ISBNEdit</tabstop>
  <tabstop>categoryEdit</tabstop>
  <tabstop>titleEdit</tabstop>
  <tabstop>publiserEdit</tabstop>
  <tabstop>yearEdit</tabstop>
  <tabstop>authorEdit</tabstop>
  <tabstop>priceEdit</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>loadButton</tabstop>
  <tabstop>addmultiButton</tabstop>
  <tabstop>ISBNEdit_2</tabstop>
  <tabstop>categoryEdit_2</tabstop>
  <tabstop>titleEdit_2</tabstop>
  <tabstop>publiserEdit_2</tabstop>
  <tabstop>authorEdit_2</tabstop>
  <tabstop>yearFromEdit</tabstop>
  <tabstop>yearToEdit</tabstop>
  <tabstop>priceFromEdit</tabstop>
  <tabstop>priceToEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>queryTableView</tabstop>
  <tabstop>borrowIDEdit</tabstop>
  <tabstop>conformButton</tabstop>
  <tabstop>borrowISBNEdit</tabstop>
  <tabstop>borrowbutton</tabstop>
  <tabstop>returnIDEdit</tabstop>
  <tabstop>conformButton_2</tabstop>
  <tabstop>returnISBNEdit</tabstop>
  <tabstop>returnButton</tabstop>
  <tabstop>borrowTableView</tabstop>
  <tabstop>returnTableView</tabstop>
  <tabstop>accountTableView</tabstop>
  <tabstop>addAccountButton</tabstop>
  <tabstop>confirmEditButton</tabstop>
  <tabstop>cancelEditButton</tabstop>
  <tabstop>deleteAccountButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>borrowIDEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>conformButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>borrowISBNEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>borrowbutton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>returnIDEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>conformButton_2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>returnISBNEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>returnButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ISBNEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>categoryEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>categoryEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>titleEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>titleEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>publiserEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>publiserEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>yearEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yearEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>authorEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authorEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>priceEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priceEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>ISBNEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ISBNEdit_2</sender>
   <signal>returnPressed()</signal>
   <receiver>categoryEdit_2</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>categoryEdit_2</sender>
   <signal>returnPressed()</signal>
   <receiver>titleEdit_2</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>titleEdit_2</sender>
   <signal>returnPressed()</signal>
   <receiver>publiserEdit_2</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>publiserEdit_2</sender>
   <signal>returnPressed()</signal>
   <receiver>authorEdit_2</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authorEdit_2</sender>
   <signal>returnPressed()</signal>
   <receiver>yearFromEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yearFromEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>yearToEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yearToEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>priceFromEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priceFromEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>priceToEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priceToEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>ISBNEdit_2</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
